### DISCUSSION TITLE
kernel

### DISCUSSION THREAD

**Discussion by Sam-hacks1** [3kh0/ext-remover#1387] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Kernel 6
version 130
board dedede
drawlat
what can I downgrade to

**Comment by schoolexploitkid** [comment]
tf
kernel 6???
if actually kernel 6 then u can downgrade to any version because it is on the wrong kernel, so it cannot restrict

**Reply by Sam-hacks1** [reply]
![Screenshot 2024-12-14 2 20 36 PM](https://github.com/user-attachments/assets/20e86e12-ebfe-4d38-89d2-4ee23c7012e6)

**Reply by schoolexploitkid** [reply]
that is ur uname, not kernver

**Comment by akabutnicer** [comment]
kernel 6 is crazy

**Reply by ghost** [reply]
lol

**Comment by Charlieis1nsan3** [comment, marked as answer]
theirs no such thing as kernel 6. just do esc+refresh+power. and then click control+D and then click tab and then find the last letter of the kernver thing.

**Reply by S-PScripts** [reply]
@Sam-hacks1 do this then i'll unmark this 'answer'
